dataland-frontend/src/components/generics/ViewFrameworkBase.vue

```diff
@@ -17,6 +17,7 @@
         <slot></slot>
       </MarginWrapper>
     </TheContent>
+    <DatalandFooter />
```

Comment:
Ich finde, wir sollten das globaler lösen statt jeder Seite den Footer zu verpassen. 
Zum Beispiel: 
* Wir packen den Footer in App.vue - mit einem v-if guard dass er nur gerendert wird, wenn die route nicht "/" ist. 
* Wir packen den Footer in App.vue als named routerView. Im Router setzen wir dann für jede Route welcher Footer verwendet werden soll

Was meinst du?

Reply:
Den Punkt sehe ich, mir ist nur erstmal keine schöne Lösung eingefallen. Named routerViews kenne ich noch gar nicht. Das schaue ich mir mal an

Reply:
Den Punkt sehe ich, mir ist nur erstmal keine schöne Lösung eingefallen. Named routerViews kenne ich noch gar nicht. Das schaue ich mir mal an.

Reply:
ok